app/Http/Controllers/Legacy/Web/CampaignsController.php

```diff
@@ -64,6 +64,7 @@ public function store(Request $request)
         $this->validate($request, [
             'internal_title' => 'required|string|unique:campaigns',
             'cause' => 'required|string',
+            'impact_doc' => 'required|string',
```

Comment:
Let's use the [`url` validator](https://laravel.com/docs/5.5/validation#rule-url) to enforce that this is a URL.

Reply:
great call!